File: SoobinJung/Lv2/최댓값과 최솟값.java
import java.util.Arrays;
import java.util.stream.Stream;

class Solution {

  public String solution(String s) {
    String words[] = s.split(" ");

    int[] values = Stream.of(words)
        .mapToInt(Integer::parseInt)
        .toArray();

    int max = Arrays.stream(values).max().getAsInt();
    int min = Arrays.stream(values).min().getAsInt();

    return min + " " + max;
  }
}

File: 권순우/LV2/최댓값과 최솟값.py
#1
def solution(s):
    nums = list(map(int,s.split()))
    maxnum = nums[0]
    minnum = nums[0]
    for i in range(len(nums)):
        newnum = nums[i]
        maxnum = max(maxnum,newnum)
        minnum = min(minnum,newnum)
    return str(minnum)+" "+str(maxnum)

#2
def solution(s):
    numarr = list(map(int,s.split(" ")))
    return str(min(numarr)) + " " + str(max(numarr))